index.html

```diff
@@ -9,7 +9,36 @@
 {% include lightbendBanner.html %}
 {% include topbar.html %}
 
-<section class="wrapper hero">
+<section class="wrapper hero-bsl">
+    <div class="row">
+        <div id="hero-bsl-left-col" class="sixcol">
+            <h2 class="bright">Do You Have Your Akka BSL License?</h2>
+            <p>A license ensures the version of Akka (beginning with v2.7 released Sept 2022) you
+                are running in production has all of the latest known bug fixes, vulnerabilities patched, and maintains
+                compliance with SOC 2 and NIST 800-53. </p>
+            <p>Licensing allows for production access to all Akka’s new features like <strong>Brokerless pub/sub</strong> and the upcoming release of
+                <strong>Akka Edge</strong> in late 2023.</p>
+            <p>More information can be found in the <a href="https://www.lightbend.com/akka/license-faq" target="_blank">BSL License FAQ</a> or <a href="mailto:akka-license@lightbend.com?subject=Akka License">contact Lightbend</a> for a License or questions. </p>
```

Comment:
```suggestion
            <p>More information can be found in the <a href="https://www.lightbend.com/akka/license-faq" target="_blank">BSL License FAQ</a> or <a href="https://www.lightbend.com/contact" target="_blank">contact Lightbend</a> for a License or questions. </p>
```
Trying to reduce spam.